test/Locators.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { browser } from 'protractor';
+import { PersonalInformationPage } from '../src/page';
+
+describe('Given I want to fill an information form', () => {
+
+  const webpage = 'http://toolsqa.com/automation-practice-form/';
+  const personalInformationPage: PersonalInformationPage = new PersonalInformationPage();
+
+  describe('When I open the webpage', () => {
+    beforeAll(async() => {
+      await browser.get(webpage);
+    });
+
+    describe('and then I fill and submit the form', () => {
```

Comment:
el step debería ser `And I fill and submit the form`

Reply:
@sorregom ya lo corregí, pero tengo una dudita: ¿Es algo estandar en el lenguaje de negocio por ejemplo ese hecho de no poner 'and then' juntos? (o simplemente mal inglés xD)